src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -130,15 +127,12 @@ static bool hasMetadata(std::string name, std::string type, ContextAttribute* ca
 
 
 
-
 /* ****************************************************************************
 *
 * equalMetadataValues -
-*
 */
-static bool equalMetadataValues(BSONObj& md1, BSONObj& md2)
+static bool equalMetadataValues(const BSONObj& md1, const BSONObj& md2)
```

Comment:
Different from the general rule with style fixing, maybe this kind of change (X& -> const X&) should be mentioned in the CNR, e.g.:

"Hardening: several changes in argument passing in mongoBackend library to align style, from X& to const X&"